hw1/02-palindrome.js

```diff
@@ -1,16 +1,24 @@
 
 const elem = document.getElementById('inputNum');
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener('input' , handleInput);
 
 
 function handleInput(elem){
     const val = document.querySelector('input').value;
-    console.log(val);
+   
     var reversedString = val.toString().split("").reverse().join("");
+    
     if (val == reversedString) {
```

Comment:
The [Control Statements](https://github.com/airbnb/javascript#control-statements) section of the airbnb guide recommends "===" rather than "==" I think === compares value and type while == only compares value